sdk/samples/aztables/go.mod

```diff
@@ -0,0 +1,11 @@
+module github.com/Azure/azure-sdk-for-go/sdk/samples/aztables
```

Comment:
We don't plant to tag/release this module right?

Reply:
No

Reply:
But I want it to be a runnable module if customers were to copy paste the code